script/release.sh

```diff
@@ -71,6 +71,7 @@ clone() {
   cd $CLI
   git remote remove origin
   git remote add origin git@github.com:$ORIGIN_REPO/$CLI.git
+  git remote add upstream git@github.com:$UPSTREAM_REPO/$CLI.git
```

Comment:
Can we maybe https instead of ssh? It will prevent accidental pushes to upstream.